### PR TITLE
centered bars

### LINKED ISSUE #6: Bars should be aligned in the center
**Is your feature request related to a problem? Please describe.**
Currently, the bars are aligned to the left.

**Describe the solution you'd like**
Bars should be aligned to the center

**Additional context**
image

### PR DESCRIPTION
**Pull request by meerhamzadev**
Fixes: #6

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#073147",
+        "titleBar.activeBackground": "#094463",
+        "titleBar.activeForeground": "#F7FCFE"
+    }
+}
```
Comment (dipanshparmar) [inline .vscode/settings.json]: Please add a new line here.

File: scss/_main.scss [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
-        background: whitesmoke;
-        height: 88vh;
-        display: flex;
+        width          : 90%;
+        margin         : auto;
+        background     : whitesmoke;
+        height         : 88vh;
+        display        : flex;
+        justify-content: center;
+        gap            : .2rem;
```
Comment (dipanshparmar) [inline scss/_main.scss]: We can remove this gap for now. It is making bars hard to see.

File: scss/_main.scss [hunk 3]
```diff
@@ -1,41 +1,43 @@
...
 
         div {
-            border: 1px solid transparent;
+            border     : 1px solid transparent;
             margin-left: 1px;
-            background: rgb(225, 186, 253);
+            background : rgb(225, 186, 253);
```
Comment (dipanshparmar) [inline scss/_main.scss]: I don't understand what is the need for aligning the colons vertically. It is causing some extra spaces. Can you please revert these colons spacing changes?

**Review by dipanshparmar — CHANGES_REQUESTED** [review]

File: scss/_main.scss (hunk 3 above)
Comment (meerhamzadev) [reply scss/_main.scss]: Oh, it's due to formater I will fix it.

File: .vscode/settings.json (hunk 1 above)
Comment (meerhamzadev) [reply .vscode/settings.json]: A new line of what can u please guide :)

Comment (dipanshparmar) [reply .vscode/settings.json]: Add a new line like this (just a simple **Enter** key click) :slightly_smiling_face: 
image

**Review by dipanshparmar — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (meerhamzadev) [reply .vscode/settings.json]: Ohhhh ok 😅

Comment (meerhamzadev) [reply .vscode/settings.json]: You can view the pull request I try to fulfill change requests.

**Review by dipanshparmar — APPROVED** [review]

**Comment by dipanshparmar** [comment]
@MeerHamza1421 Thanks for the PR :smile:

**Pull request merged**